backends/bmv2/jsonconverter.cpp

```diff
@@ -1844,11 +1849,13 @@ void JsonConverter::addMetaInformation() {
   auto meta = new Util::JsonObject();
 
   static constexpr int version_major = 2;
-  static constexpr int version_minor = 4;
+  static constexpr int version_minor = 6;
   auto version = mkArrayField(meta, "version");
   version->append(version_major);
   version->append(version_minor);
 
+  meta->emplace("compiler", "https://github.com/p4lang/p4c");
```

Comment:
my reasoning is that there is only one p4c backend for bmv2; anyway this has no real use right now except providing the ability to quickly see which compiler produced the json (old one or new one)